src/vis_cpu/conversions.py

```diff
@@ -22,3 +25,149 @@ def lm_to_az_za(el, m):
     az = -np.arctan2(m, el)
     za = np.pi / 2.0 - np.arcsin(n)
     return az, za
+
+
+def point_source_crd_eq(ra, dec):
+    """
+    Calculate approximate coordinate transform of source locations from 
+    equatorial to Cartesian.
+    
+    Parameters
+    ----------
+    ra, dec : array_like
+        1D arrays of source positions in equatorial coordinates (radians).
+    
+    Returns
+    -------
+    array_like
+        Equatorial coordinates of Healpix pixels, in Cartesian
+        system. Shape=(3, NPIX).
+    """
+    return np.asarray([np.cos(ra)*np.cos(dec), np.cos(dec)*np.sin(ra),
+                     np.sin(dec)])
+
+
+def uvbeam_to_lm(uvbeam, freqs, n_pix_lm=63, polarized=False, **kwargs):
+    """
+    Convert a UVbeam to a uniform (l,m) grid
+
+    Parameters
+    ----------
+    uvbeam : UVBeam object
+        Beam to convert to an (l, m) grid.
+    
+    freqs : array_like
+        Frequencies to interpolate to in [Hz]. Shape=(NFREQS,).
+    
+    n_npix_lm : int, optional
+        Number of pixels for each side of the beam grid. Default is 63.
```

Comment:
I don't like having defaults in the docstring unless they are unclear from the signature itself. I'd remove this and the next.

Reply:
Done.